src/theme/styleVars.ts

```diff
@@ -24,7 +24,8 @@ const colors = {
   white80a: 'rgba(255, 255, 255, 0.8)',
   yellow: '#EFC93D',
   orange: '#E17B60',
-  mint: '#3EB489' // TODO: add the right hex value here
+  mint: '#3EB489',
+  cornflower: '#4C7180' // TODO: add the right hex value here
```

Comment:
We probably don't need this comment anymore - I don't remember why we had it lol